### ISSUE TITLE
Psionics Coercion Mind Reading Character Limited

### ISSUE THREAD

**Issue by Menkhu** [HearthOfHestia/Nebula#205]
#### Description of issue

Replies to coercion mind reading questions seem to be severely limited by length. They also don't give any indicator to the mind reader or the person being read when the length is too long. The answers seems to be submitted by the person being read, and the mind reader receives the normal 'You receive nothing useful' message which is usually given when the person being read refuses to answer.

#### Difference between expected and actual behavior

I can't pin down the exact character limit, but it seems to be very small and impossible to ascertain when actually writing a response to a mind-reading question. 

#### Steps to reproduce

In game, try to read someones mind with the coercion psionics faculty. Certain reply lengths will not be submitted as described above.

#### Length of time in which bug has been known to occur

Noticed 28/11/2021 during the scheduled high pop testing round.

#### Client version, Server revision & Game ID

Unknown (I forgot to check at the time, sorry)

#### Issue bingo

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)

**Comment by MistakeNot4892** [comment]
I don't believe that length should break input() like this - there are no length checks or sanitization. That failure message is also printed if you take more than sixty seconds to reply, if the user is unconscious, or the target dies. I'll slap in a commit to change `as text` to `as message` in case that helps though.

**Comment by Menkhu** [comment]
> I don't believe that length should break input() like this - there are no length checks or sanitization. That failure message is also printed if you take more than sixty seconds to reply, if the user is unconscious, or the target dies. I'll slap in a commit to change `as text` to `as message` in case that helps though.

Oddly enough the window didn't seem to close in sixty seconds, but the replies were still ignored when submitted in favour of the 'You receive nothing useful' message. I didn't time it though so I could be wrong here. I'll try to check next round I can.

**Comment by MistakeNot4892** [comment]
We can't make BYOND close an input() window, hence the sixty seconds check.

**Comment by Menkhu** [comment]
Might be reasonable to extend the check then? Almost every time I've tried to use the ability, the person on the receiving end hasn't been able to get their replies in on time, and they haven't been aware their messages weren't actually getting through, which leads to a lot of confusion.
